app/models/site_config.rb

```diff
@@ -12,6 +12,7 @@ class SiteConfig < RailsSettings::Base
 
   STACK_ICON = File.read(Rails.root.join("app/assets/images/stack.svg")).freeze
   LIGHTNING_ICON = File.read(Rails.root.join("app/assets/images/lightning.svg")).freeze
```

Comment:
non-blocker: oooh, it would be great if we could create a helper in `URL` for these, too. i'm not a big fan of us doing `File.read` directly within the `SiteConfig`

Reply:
These are actually SVGs - the file contents itself go into the field so we need `File.read`
![Screen Shot 2020-10-28 at 11 46 26 AM](https://user-images.githubusercontent.com/15987080/97460565-405f6a00-1913-11eb-87f7-017e6a3d7def.png)